packages/frontend/src/components/RefreshButton.tsx

```diff
@@ -17,11 +17,8 @@ export const RefreshButton = memo(() => {
     const fetchResults = useExplorerContext(
         (context) => context.actions.fetchResults,
     );
-    const hasUnfetchedChanges = useExplorerContext(
-        (context) => context.hasUnfetchedChanges,
-    );
 
-    const canRunQuery = !isLoading && isValidQuery && hasUnfetchedChanges;
+    const canRunQuery = !isLoading && isValidQuery;
```

Comment:
I don't think we need this variable here anymore,

we should also remove it from line # 47